public/app/features/alerting/unified/components/rule-editor/VizWrapper.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC, useState } from 'react';
+import AutoSizer from 'react-virtualized-auto-sizer';
+import { css } from '@emotion/css';
+import { GrafanaTheme, PanelData, VizOrientation } from '@grafana/data';
+import { config, PanelRenderer } from '@grafana/runtime';
+import { LegendDisplayMode, SingleStatBaseOptions, TooltipDisplayMode, RadioButtonGroup, useStyles } from '@grafana/ui';
+
+interface Props {
+  data: PanelData;
+  defaultPanel?: 'timeseries' | 'table' | 'stat';
+}
+
+export const VizWrapper: FC<Props> = ({ data, defaultPanel }) => {
+  const [pluginId, changePluginId] = useState<string>(defaultPanel ?? 'timeseries');
+  const options = { ...getOptionsForPanelPlugin(pluginId) };
+  const styles = useStyles(getStyles);
```

Comment:
nit: we're moving away from GrafanaTheme and useStyles to GrafanaTheme2 and useStyles2. I think you should too.